docs/rules/valid-local-dependency.md

```diff
@@ -0,0 +1,25 @@
+# Ensures that local dependencies specified in a package.json exist
+
+## Rule Details
+
+This rule validates the casing for `file:` and `link:` dependencies in a `package.json` file. If the casing of a `link:` or `file:` path is incorrect yarn does not sub out the link when releasing.
```

Comment:
[Docs] It validates more than casing, no? Like, if the file doesn't exist at all, this will catch that, right?

Maybe:

```suggestion
This rule validates the path for `file:` and `link:` dependencies in a `package.json` file - including name casing. If a `link:` or `file:` path is incorrect yarn does not sub out the link when releasing.
```